src/tencent/credential.rs

```diff
@@ -115,7 +115,18 @@ impl CredentialLoader {
             (Some(region), Some(token_file), Some(role_arn), Some(provider_id)) => {
                 (region, token_file, role_arn, provider_id)
             }
-            _ => return Ok(None),
+            _ => {
+                debug!(
+                    "assume_role_with_web_identity is not configured fully, region: {region:?}, \
```

Comment:
It's better not to print out the value, we can print while value is missing instead.